### PR TITLE
implemented SVC model using GridSearchCV #6

### PR DESCRIPTION
**Pull request by muskan191103**
- Used OneHotEncoder for encoding 'type' column.
- Checked for NAN values and filled them with 0 and mean wherever required.
- Used train_test_split method for dividing the dataset into training and testing dataset.
- Used GridSearchCV for checking the best parameters for SVC model.
Screenshot 2024-03-16 225422

### CODE REVIEW

**Comment by Samcoding5854** [comment]
@muskan191103 

Implement more models other than SVC in gridsearchCV and also show the accuracy they have achieved

**Comment by Samcoding5854** [comment]
Could have used more models to compare accuracy rather than just parameter tuning the SVC model.
Rest, perfect implementation of the code and grid searchCV.
